src/Worker/Handler/DebugTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare (strict_types=1);
+
+namespace Maleficarum\Worker\Handler;
```

Comment:
I don't think this is the correct namespace for this trait - it's not a trait for all Handlers, it only makes sense when using the debug encapsulator.

Reply:
Changed at 5f3f302cb54de38384d4ba0180bb30fabe25ec7e